source/index.html.md

```diff
@@ -5230,7 +5232,7 @@ func main() {
 
 `GET /payment_requests/outgoing`
 
-Payment Requests where you're the creditor and collecting funds from the debtor. Alternatively the debtor is sending you funds ([Receivable Contacts](/#add-a-receivable-contact)) <aside class="notice"> The **/payment_requests/outgoing** endpoint will continue to be supported. </aside>
+Payment Requests where you're the creditor and collecting funds from the debtor. <aside class="notice"> The **/payment_requests/outgoing** endpoint will be supported until 31 June 2021. Any integrations to this endpoint should be pointed at the new **/payment_requests/collections** endpoint which returns the same information. </aside>
```

Comment:
Maybe?
```suggestion
Payment Requests where you're the creditor and are _collecting_ funds from the debtor. <aside class="notice"> The **/payment_requests/outgoing** endpoint will be supported until 31 June 2021. Any integrations to this endpoint should be pointed at the new **/payment_requests/collections** endpoint which returns the same information. </aside>
```

Reply:
Strictly speaking, `collections` won't return the same information (since it won't include receivables). Is it worth explaining this change a little? (or linking somewhere, eg a change log)

Reply:
I hear ya. Only 2 merchants are using the old /outgoing endpoint and both are for collections only. 
Any new integrators should see the Deprecated warning and move straight onto the Collections or Receivable endpoints, so I just wanted to keep it super simple to avoid any confusion for the 2 existing integrations.